test/extended/cli/version.go

```diff
@@ -0,0 +1,106 @@
+package cli
+
+import (
+        "encoding/json"
+        "fmt"
+        "log"
+        "os"
+        "regexp"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cli] oc [Slow]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc                     = exutil.NewCLI("oc", exutil.KubeConfigPath())
+	)
+
+	g.It("Critical-30285-Checking oc version before login", func() {
```

Comment:
Please add [Slow] here. "Critical-30285-Checking oc version before login" --> "Critical-30285-Checking oc version before login [Slow]"

Reply:
@kuiwang02 Thanks , have updated , please help review again.

Reply:
@zhouying7780 it is good for me to move test label Slow. thanks

Reply:
@kuiwang02 Since the case will update KUBECONFIG and ~/.kube/config, so need to run by sequentially, I removed the [Slow] label and add [Serial]